docs/topics/cli-run.adoc

```diff
@@ -11,11 +11,11 @@ You can run {ProductShortName} against your application.
 .Procedure
 
 . Open a terminal and navigate to the `<{ProductShortName}_HOME>/bin/` directory.
-. Execute the `windup-cli` script, or `windup-cli.bat` for Windows, and specify the appropriate arguments:
+. Execute the `{mta-cli}` script, or `{mta-cli}.bat` for Windows, and specify the appropriate arguments:
```

Comment:
In Windows, the mta-cli is an `.exe` file not a `.bat`

Reply:
yes, that is fixed in https://github.com/windup/windup-documentation/pull/828